lib/resources/r4/observation.yaml

```diff
@@ -510,6 +884,114 @@ fields:
     </ul>
   url: https://hl7.org/fhir/R4/observation-definitions.html#Observation.referenceRange.type
 
+- name: referenceRange
+  required: 'No'
+  type: List of BackboneElement
+  description: Provides guidance for interpretating the observation.
+  action: update
+  example: |
+    {
+      "referenceRange": [
+        {
+          "low": {
```

Comment:
should have lab examples here too.

Reply:
do we need to say that we do not support critical reference ranges? I don't remember the final decision

Reply:
Since we 422 if a referenceRange.type other than 'normal' is supplied, we should mention that. Something like, "Only a referenceRange.type of 'normal' is supported." This would need to be added to 'create' section as well.

Reply:
+1

Reply:
updated: https://github.com/cerner/fhir.cerner.com/commit/24cb43e7f384892b8a8dbb1aaf51ec9703f2b949#diff-e303aa1b2a83c6a68a12604d4956dcd3c31f4ffe40546bbcc31e2cc6588921dcR905-R915

Reply:
@Manjushavrk still not seeing the implementation note for reference range type field. this would require update to create and update for ReferenceRange.type field

Reply:
updated notes for create and update : https://github.com/cerner/fhir.cerner.com/commit/61a5762329d4668f975d003c578de77e2e8f3e4c#diff-e303aa1b2a83c6a68a12604d4956dcd3c31f4ffe40546bbcc31e2cc6588921dcR824
https://github.com/cerner/fhir.cerner.com/commit/61a5762329d4668f975d003c578de77e2e8f3e4c#diff-e303aa1b2a83c6a68a12604d4956dcd3c31f4ffe40546bbcc31e2cc6588921dcR932